scripts/apply-patch.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# If you pass patch file as first parameter this script apply patch to just
+# particular file.
+
+# If you don't provide any parameter:
+# This script look for all .patch files in ocs-ci repo and trying to download
```

Comment:
Replace:
'look' -> 'looks'
'and trying' -> 'and tries'

Reply:
done